lib/pipeline.ts

```diff
@@ -398,6 +398,13 @@ export class Pipeline extends Construct {
     }), options);
   }
 
+  public publishToGolang(options: publishing.PublishToGolangProps) {
```

Comment:
That name is... awkward. Other functions are named `publishTo<PackageRegistry>`, this one is `publishTo<Language>`. I guess it may be better off named `publishGoBindingsToGitRepository` or something...

Not exactly important, though.

Reply:
I'll leave like this. We use `golang` when it comes to publishing go bindings to github in jsii-release (and projen) so I wanted  to stay consistent.

Reply:
Added a docstring with some details.